docs_src/getting-started/tools/Ch-CommandLineInterface.md

```diff
@@ -1,104 +1,714 @@
+### `edgex-cli deviceservice add` 
+Add a device service
 
-**For more information read, see the tool's [README](https://github.com/edgexfoundry/edgex-cli/blob/master/README.md).**
+```
+$ edgex-cli deviceservice add -n TestDeviceService -b "http://localhost:51234"
+```
 
-## What can the CLI do?
+### `edgex-cli deviceservice name` 
+Shows information about a device service. Most `edgex-cli` commands support the `-v/--verbose` and `-j/--json` flags:
 
-The Edgex CLI covers most of the basic EdgeX Foundry APIs. Using the CLI, users can query all the available devices, profiles, services, events, readings etc. or query for a subset of them based on available flags – for example get devices with a specific name or get all events that belong to specific device.
+```
+$ edgex-cli deviceservice name -n TestDeviceService
+Name               BaseAddress             Description
+TestDeviceService  http://localhost:51234  
+
+$ edgex-cli deviceservice name -n TestDeviceService -v
+Name               BaseAddress             Description  AdminState  Id                                    Labels  LastConnected  LastReported  Modified
+TestDeviceService  http://localhost:51234               UNLOCKED    7f29ad45-65dc-46c0-a928-00147d328032  []      0              0             10 Jan 22 17:26 GMT
+
+$ edgex-cli deviceservice name -n TestDeviceService  -j | jq '.'
+{
+  "apiVersion": "v2",
+  "statusCode": 200,
+  "service": {
+    "created": 1641835585465,
+    "modified": 1641835585465,
+    "id": "7f29ad45-65dc-46c0-a928-00147d328032",
+    "name": "TestDeviceService",
+    "baseAddress": "http://localhost:51234",
+    "adminState": "UNLOCKED"
+  }
+}
+```
 
-It also provides the means to create EdgeX objects such as devices, profiles, etc.  Creating can be accomplished in three different ways:  
-- By providing a file that contains the normal create object API request body
-- By using inline parameters
-- Or using interactive mode.
+### `edgex-cli deviceservice rm` 
+Remove a device service
 
-Some of the request bodies contain a lot of parameters and specifying them in one single line is not convenient.  For these cases, using the interactive mode with or without inline parameters parameters would be more convenient.
+```
+$ edgex-cli deviceservice rm -n TestDeviceService
+```
 
+### `edgex-cli deviceservice update` 
+Update the device service, getting the ID using jq and confirm that the labels were added
 
-Updating EdgeX objects via the CLI can done in two ways:
- - By providing a file that contains the update API request body that would normally be used in the API call
- - Or by providing just the name of the object.  If the object with such name exists, the CLI opens an editor with a preloaded request body from the retrieved values, allowing you to change/edit desired fields (except the name and ID). 
+```
+$ ID=`./bin/edgex-cli deviceservice name -n TestDeviceService -j | jq -r '.service.id'`
```

Comment:
I am using edgex-cli snap and got `bash: ./bin/edgex-cli: No such file or directory`